Source/SwiftLintFramework/Rules/NoFallthroughRuleOnlyRule.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import SourceKittenFramework
+
+public struct NoFallthroughOnlyRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "no_fallthrough_only",
+        name: "No Fallthrough Only",
+        description: "Fallthroughs can only be used if the `case` contains at least one other statement.",
+        kind: .idiomatic,
+        nonTriggeringExamples: [
+            """
+            switch {
+            case 1:
+                var a = 1
+                fallthrough
+            case 2:
+                var a = 2
+            }
+            """,
+            """
+            switch {
+            case "a":
+                var one = 1
+                var two = 2
+                fallthrough
+            case "b": /* comment */
+                var three = 3
+            }
+            """,
+            """
+            switch {
+            case 1:
+               let one = 1
+            case 2:
+               // comment
+               var two = 2
+            }
+            """
+        ],
+        triggeringExamples: [
+            """
+            switch {
+            case 1:
+                fallthrough
```

Comment:
can you add the violation marks (`↓`) to the examples?

Reply:
How are those marks used exactly? Do you place them at the leading or trailing edge of the violation?

Reply:
You should place them where the violation is reported. In this case, as an user, I'd expect the violation to be triggered on `↓fallthrough`, but reading the code I believe it'll be triggered on `↓case 1:`. 

Adding the marker also makes the unit tests to verify that the violation is being reported on the expected offset.